frontend/src/pages/WhatIsHealthEquity/WhatIsHealthEquityPage.module.scss

```diff
@@ -331,42 +331,14 @@
     padding: 10px 0 10px 0;
   }
 
-  .EmailAddressForm {
-    border-radius: 40px;
-    width: 485px;
-    height: 60px;
-    background-color: white;
-    border: 1px solid #888888;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-between;
-  }
-
-  .EmailAddressFormText {
-    height: 60px;
-    background-color: transparent;
-    border: none;
-    margin-left: 20px;
-    font-size: 16px;
-
-    &:focus {
-      outline: none;
-    }
+  .EmailTextField {
+    width: 260px;
   }
 
   .EmailAddressFormSubmit {
```

Comment:
I also think we want to maintain this positioning inside of the outlined form per the mocks as well